charts/cloudnative-pg/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: cloudnative-pg
 description: CloudNativePG Operator Helm Chart
 icon: https://raw.githubusercontent.com/cloudnative-pg/artwork/main/cloudnativepg-logo.svg
 type: application
-version: "0.21.2"
+version: "0.22.0"
```

Comment:
Could you drop this change. We bump the version number as part of our release process. Moreover, given that this is a non-breaking change, a minor release would suffice.